src/cwe_checker_lib/src/checkers/cwe_190.rs

```diff
@@ -19,15 +20,16 @@
 //! - There is no check whether the result of the multiplication is actually used
 //!   as input to the function call. However, this does not seem to generate a lot
 //!   of false positives in practice.
-//! - There is no value set analysis in place to determine whether an overflow is
-//!   possible or not at the specific instruction.
+//! - Relative values are always generate a hit.
```

Comment:
The reader might not know about the intrinsics of the cwe_checker and thus might not know what a relative value is in this context.